File: python-hello/src/main/python/FizzBuzz.py
# -*- coding=utf-8 -*-
import sys

n = int(sys.argv[1])
print(n)

for i in range(1, n + 1):
  整除5 = i % 5 == 0
  整除3 = i % 3 == 0
  if 整除3 and 整除5:
      print('FizzBuzz')
  elif 整除3:
      print("Fizz")
  elif 整除5:
      print("Buzz")
  else:
      print(i)

File: perl-hello/FizzBuzz.pl
#!/usr/bin/perl
$n = @ARGV[0];
for( $i = 1; $i <= 100; $i = $i + 1){
  if($i % 3 ==0 && $i % 5==0){
    print "FizzBuzz\n";
  }elsif($i%3==0){
    print "Fizz\n";
  }elsif($i % 5 ==0){
    print "Buzz\n";
  }else{
    print "$i\n";
  }
}